examples/30_extended/tasks_tutorial.py

```diff
@@ -73,24 +73,21 @@
 #
 # Similar to listing tasks by task type, we can list tasks by tags:
 
-tasks = openml.tasks.list_tasks(tag='OpenML100')
-tasks = pd.DataFrame.from_dict(tasks, orient='index')
+tasks = openml.tasks.list_tasks(tag='OpenML100', output_format='dataframe')
```

Comment:
I think we should add this in the `tasks_tutorial.py`:
```
#   * Use the output_format parameter to select output type
#   * Default gives 'dict' (other option: 'dataframe', see below)
```

https://github.com/openml/openml-python/blob/6582cbd52e84c10811c6eb303401c9101a252e6b/examples/30_extended/tasks_tutorial.py#L34-L36

The above makes it seem like there is only one return type (a dictionary of dictionaries) while the return type is controlled by the `output_format`.

Reply:
I rephrased the example. I hope this makes it a bit better.

Reply:
(the actual information you wanted was already there, but just not as easily visible as for the dataset tutorial).

Reply:
> (the actual information you wanted was already there, but just not as easily visible as for the dataset tutorial).

No, I found it, however I still thought it would be confusing for a new user.

Reply:
> I rephrased the example. I hope this makes it a bit better.

Yes, it looks way better to me now.